src/hooks/useRequestTokenAndSendMessage.ts

```diff
@@ -8,7 +8,7 @@ export default function useRequestTokenAndSendMessage() {
 
   const { mutateAsync: requestToken } = useRequestToken()
   const { mutateAsync: sendMessage } = useSendMessage()
-  const login = useMyAccount((state) => state.login)
+  const login = useMyAccount((state) => state.loginAnonymously)
```

Comment:
maybe need to rename this to loginAnonymously to be more specific

Reply:
Yes, could do!